parts/linux/cloud-init/artifacts/kubelet.service

```diff
@@ -26,7 +26,7 @@ ExecStart=/usr/local/bin/kubelet \
         --node-labels="${KUBELET_NODE_LABELS}" \
         --v=2 {{if NeedsContainerd}}--container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix:///run/containerd/containerd.sock{{end}} \
         --volume-plugin-dir=/etc/kubernetes/volumeplugins \
-        {{- if IsDynamicKubeletEnabled}}
+        {{- if IsKubeletConfigFileEnabled}}
         --config /etc/default/kubeletconfig.json \
         {{- end}}
         $KUBELET_FLAGS \
```

Comment:
Just curious , can we combine --config /etc/default/kubeletconfig.jso into KUBELET_FLAGS? I think we don't have a clear principal as where the kubelet flags should go

Reply:
Technically we can, but I just want to call it out specifically here.

Reply:
got you.